chain/consensus/mir/types.go

```diff
@@ -239,7 +226,7 @@ func UnwrapCheckpointSnapshot(ch *checkpoint.StableCheckpoint) (*Checkpoint, err
 	return snap, err
 }
 
-// Get stable checkpoint by height from datastore.
+// GetCheckpointByHeight stable checkpoint by height from datastore.
```

Comment:
sanity-check: this is no longer used?

Reply:
That one is still used. I did not touch it

Reply:
Gotcha! I didn't see it in the files changes and I didn't know if by any change it wasn't used anywhere.